wait-for-manager.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+# wait-for-manager.sh
+
+set -e
+
+until test -f /healthcheck/manager-ready ; do
+  >&2 echo "Manager is not ready - sleeping"
+  sleep 1
+done
+
+>&2 echo "Manager is up - starting worker"
+
```

Comment:
giving a link (to docker-entrypoint.sh) and explaining below line, we can merge the pr